README.md

```diff
@@ -420,6 +420,23 @@ plugin, depending on its state. This command accepts an optional argument
 ### `plug`
 And last but not least: `plug`. Load plugin from plugin installation directory by its name.
 
+## Tips and Tricks
+
+### Automatic Installation
+
+In your kakrc, replace the ```source "%val{config}/plugins/plug.kak/rc/plug.kak"``` line with the following:
+
+```kak
+try %{
+  source "%val{config}/plugins/plug.kak/rc/plug.kak"
+} catch %{ nop %sh{
+  if [ ! -d "$kak_config/plugins/plug.kak" ]; then
+    git clone "https://github.com/robertmeta/plug.kak.git" "$kak_config/plugins/plug.kak"
+    echo "source '%val{config}/plugins/plug.kak/rc/plug.kak'"
+  fi
+}}
```

Comment:
```suggestion
try %{
    source "%val{config}/plugins/plug.kak/rc/plug.kak"
} catch %{ nop %sh{
    if [ ! -d "$kak_config/plugins/plug.kak" ]; then
        git clone "https://github.com/robertmeta/plug.kak.git" "$kak_config/plugins/plug.kak"
        echo "source '%val{config}/plugins/plug.kak/rc/plug.kak'"
    fi
}}
```
I think we should keep the same indentation as the other parts.

Reply:
Fair enough - consistency would be rather nice, wouldn't it?